content/en/docs/reference/glossary/cluster.md

```diff
@@ -4,20 +4,19 @@ id: cluster
 date: 2019-06-15
 full_link: 
 short_description: >
-   A set of worker machines, called nodes, that run containerized applications. Every cluster has at least one worker node.
-
+   A set of machines hosting containerized workloads, in the form of pods, and the control plane that manages them.  
 aka: 
 tags:
 - fundamental
 - operation
 ---
-A set of worker machines, called {{< glossary_tooltip text="nodes" term_id="node" >}},
-that run containerized applications. Every cluster has at least one worker node.
+A set of machines hosting containerized {{< glossary_tooltip text="workloads" term_id="workload" >}}, in the form of {{< glossary_tooltip text="pods" term_id="pod" >}}, and the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} that manages them.  
```

Comment:
I think we can do better. Try this:
```suggestion
A {{< glossary_tooltip text="control plane" term_id="control-plane" >}} and a set of worker machines that the
control plane manages, with the aim of running containerized {{< glossary_tooltip text="workloads" term_id="workload" >}}
as {{< glossary_tooltip text="pods" term_id="pod" >}}.
```

Reply:
This depends on the other comment...

Reply:
Line 14 seems to be missing something?

Reply:
I revised. BTW you can accept suggestions as is, but you can also incorporate the spirit of the feedback and apply your own edits.